tasks/libs/common/git.py

```diff
@@ -45,6 +45,40 @@ def get_staged_files(ctx, commit="HEAD", include_deleted_files=False) -> Iterabl
                 yield file
 
 
+def get_file_modifications(
+    ctx, base_branch=DEFAULT_BRANCH, added=False, modified=False, removed=False, only_names=False
+) -> list[tuple[str, str]]:
+    """
+    Get file status changes for the current branch compared to the base branch.
+
+    If no filter is provided, will return all the files.
+
+    - added: Include added files
+    - modified: Include modified files
+    - removed: Include removed files
+    - only_names: Return only the file names without the status
+    - Returns a list of (status, filename)
```

Comment:
🥜 We could probably use a more uniform layout in our [docstring](https://peps.python.org/pep-0257/#multi-line-docstrings)

Reply:
*Taking notes*